README.rdoc

```diff
@@ -2,12 +2,16 @@
 
 1. Clone it
 
-2. rake db:migrate
+2. Rename database.yml.example to database.yml (or customize)
 
-3. rake db:seed
+3. Rename secrets.yml.example to secrets.yml (or generate a new one)
 
-4. rails s
+4. rake db:setup
```

Comment:
If you want to do setup you dont need the rake db:seed.  

I would just leave it as rake db:migrate